enterprise/backend/src/metabase_enterprise/semantic_search/db/util.clj

```diff
@@ -0,0 +1,18 @@
+(ns ^{:clj-kondo/ignore [:discouraged-var]}
```

Comment:
why? can we tag it on the relevant callsite directly for clarity instead of the whole namespace?